### DISCUSSION TITLE
Container tag defined incorrectly

### DISCUSSION THREAD

**Discussion by kleiner-sentry** [FakeFishGames/Barotrauma#13838] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I loaded into the game, and got a popup in the console saying that wreckchaingunloader was defined incorrectly

### Reproduction steps

1. load into the game
2. look at console

### Bug prevalence

Just once

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
[Vanilla] Container tag "wreckchaingunloader" defined in ContainerTagPrefab is not used in any item prefabs, did you misspell it?
```

**Comment by Regalis11** [comment]
Thank you for the report! This error essentially means that some of the pre-defined container tags the game allows assigning through the sub editor's "tag picker" don't exist in any item. It's most likely caused by mods overriding some vanilla items and changing their tags. I changed this to a warning instead of an error now, because it shouldn't cause any actual issues, and may even be intentional in some cases. Commit in our private repo: https://github.com/FakeFishGames/Barotrauma-development/commit/ad8887516fa227bebb441b16dab576f817d149f5.
